code/modules/mob/living/silicon/robot/robot_damage.dm

```diff
@@ -4,8 +4,15 @@
 		stat = CONSCIOUS
 		return
 	health = maxHealth - (getBruteLoss() + getFireLoss())

```

Comment:
I found two, and wasn't sure which of them to keep. I'll keep the other one, then.